src/pyedb/dotnet/edb_core/cell/primitive/primitive.py

```diff
@@ -62,6 +60,14 @@ def __init__(self, pedb, edb_object):
             "rectangle": bondwire_cross_section_type.BondwireRectangle,
         }
 
+    @property
+    def _core_stackup(self):
+        return self._app.stackup
+
+    @property
+    def _core_net(self):
+        return self._app.nets
+
```

Comment:
I suggest to remove these two properties

self._core_stackup is only used in layer_name()

self._core_net is never used.